docs/guides/custom-functions.md

```diff
@@ -126,6 +130,7 @@ module.exports = (targetVal, opts) => {
 If for some reason, you do not want to place your functions in a directory called `functions`, you can specify a custom directory.
 
 ```yaml
+functions: [abc]
```

Comment:
@Snoopy1964 Does this fit your request from #868?

Reply:
@nulltoken Yup, the correction you made in the example section of the custom-functions.md in the master branch fits!

Nevertheless I can't see it in the official documentation https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/guides/custom-functions.md#example

Thanks a lot!

Reply:
@philsturgeon Would you know how often is the main website updated with the doco content? It seems to still reflect the old content.

![image](https://user-images.githubusercontent.com/92363/72371416-a1aba900-3704-11ea-9ab6-80c30dca7e4b.png)